src/consumer/component/sources/files.js

```diff
@@ -54,12 +52,12 @@ export default class Files extends Source {
         return Promise.resolve();
       }
       const filePath = path.join(projectRoot, bitMapFiles[file.name]);
-      return this.writeOneFile(filePath, file.content.contents, force);
+      return this.writeOneFile(filePath, file.contents, force);
```

Comment:
Everything now is one file, so please remove the writeOneFile method, and move its code into write.